src/governance-vote.h

```diff
@@ -197,6 +207,8 @@ class CGovernanceVote
 };
 
 
+bool operator==(const CGovernanceVote& vote1, const CGovernanceVote& vote2);
```

Comment:
Because it's excessive? You already defined `==` operator for `CGovernanceVote` on line#94, why would you define it here again?